boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,161 @@
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/lpcxpresso54114/lpcxpresso54114_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC54114 SoC has IOCON registers which can be used to configure the
+functionality of a pin. Following pins are configured to support the
+basic blinky and hello world sample applications. 
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_0  | USART           | USART RX                   |
++---------+-----------------+----------------------------+
+| PIO0_1  | USART           | USART TX                   |
++---------+-----------------+----------------------------+
+| PIO0_29 | GPIO            | RED LED                    |
++---------+-----------------+----------------------------+
+| PIO1_10 | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The LPC54114 SoC is configured to use internal FRO at 48MHz as a source for 
+the system clock. Many other sources for system clock is provided in soc which 
+can be used as per the requirement.
+
+Serial Port
+===========
+
+The LPC54114 SoC has 8 FLEXCOMM interfaces for serial communication. 
+One is configured as USART for the console and the remaining are not used.
+
+Programming and Debugging
+*************************
+
+Segger J-Link tools have been used here for debug purpose.
+Set ``DEBUG_SCRIPT = jlink.sh`` in Makefile.Board.
+
+Debugging
+=========
+
+This example uses the :ref:`hello_world` sample with the jlink script.
+Use ``make BOARD=lpcxpresso54114 debug`` build target to build your Zephyr 
+application, invoke the jlink GDB server, attach a GDB client, and program your 
+Zephyr application to flash. It will leave you at a gdb prompt.
+
+.. code-block:: console
+
+   $ cd <zephyr_root_path>
+   $ . zephyr-env.sh
+   $ cd samples/hello_world/
+   $ make BOARD=lpcxpresso54114 debug
+
+Open a serial terminal (minicom, putty, etc.) and connect board with the following settings:
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Reset the board and you should be able to see on the corresponding Serial Port
+the following message:
+
+.. code-block:: console
+
+   Hello World! arm
+
+
+.. _LPC54114 SoC Website:
+	http://www.nxp.com/products/microcontrollers-and-processors/arm-processors/lpc-cortex-m-mcus/
```

Comment:
this is the actual url and will be too long for single line..should i still put that in single line ?